client/packages/android/capacitor.config.ts

```diff
@@ -5,7 +5,7 @@ import { CapacitorConfig } from '@capacitor/cli';
 
 const config: CapacitorConfig = {
   appId: 'org.openmsupply.client',
-  appName: 'openmsupply-client',
+  appName: 'Open mSupply',
```

Comment:
it was the change in stirngs.xml which made it work, this was done for completeness